### PR TITLE
SNOW-514303 Support n:1 topics:table

### PR DESCRIPTION
**Pull request by sfc-gh-zefan**
Support multiple topics mapping to one table by deleting the check in the configuration.

### CODE REVIEW

File: test/rest_request_template/travis_correct_multiple_topic_to_one_table_snowpipe.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "SNOWFLAKE_CONNECTOR_NAME",
+  "config": {
+    "connector.class": "com.snowflake.kafka.connector.SnowflakeSinkConnector",
+    "snowflake.role.name": "TESTROLE_KAFKA",
+    "topics": "SNOWFLAKE_TEST_TOPIC",
+    "snowflake.topic2table.map": "NUM_TOPICS<3>NUM_TOPICS",
```
Comment (sfc-gh-tzhang) [inline test/rest_request_template/travis_correct_multiple_topic_to_one_table_snowpipe.json:7]: could you explain what is `NUM_TOPICS<3>NUM_TOPICS`? My understanding is that we need to provide a map here, so something like `topic1:table,topic2:table`

Comment (sfc-gh-zefan) [reply test/rest_request_template/travis_correct_multiple_topic_to_one_table_snowpipe.json:7]: Enclosed in <> is the number of topics to be generated. I added this to make the number of topics both configurable and compatible to the old behavior of the test script.

Comment (sfc-gh-zefan) [reply test/rest_request_template/travis_correct_multiple_topic_to_one_table_snowpipe.json:7]: It is basically equivalent to `topic1:table,topic2:table,topic3:table`, except that the name of the table and topics are not hard-coded.

**Review by sfc-gh-tzhang — APPROVED** [review]
LGTM!

File: src/test/java/com/snowflake/kafka/connector/ConnectorConfigTest.java [hunk 2]
```diff
@@ -127,7 +127,7 @@ public void testDuplicatedTopic() {
     Utils.validateConfig(config);
   }
 
-  @Test(expected = SnowflakeKafkaConnectorException.class)
+  @Test()
```
[the remark this answers is not in the document]
Comment (sfc-gh-zefan) [reply src/test/java/com/snowflake/kafka/connector/ConnectorConfigTest.java]: done

**Pull request merged**